src/instructlab/model/evaluate.py

```diff
@@ -187,6 +163,32 @@ def get_evaluator(
         )
 
 
+def validate_model(model: str, model_arg: str = "--model"):
+    if os.path.exists(model):
+        model_path = pathlib.Path(model)
+        valid_model = False
+        if model_path.is_dir():
+            valid_model = backends.is_model_safetensors(model_path)
+        elif model_path.is_file():
+            valid_model = backends.is_model_gguf(model_path)
+        if not valid_model:
+            click.secho(
+                f"Evaluate '{model_arg}' needs to be passed either a safetensors directory or a GGUF file",
+                fg="red",
+            )
+            raise click.exceptions.Exit(1)
+        click.secho(
+            f"Using local model found at '{model_path}' for '{model_arg}'",
+            fg="blue",
+        )
+    else:
+        click.secho(
+            f"Model could not be found at '{model}'",
```

Comment:
Could you change this to `Model could not be found at '{model]' for '{model_args}'.

Reply:
Changed, thanks!